db/inmemory/client.go

```diff
@@ -4,6 +4,7 @@ import(
 	"github.com/go-swagger/go-swagger/examples/ShivaniCustomServerExample1/domain"
 	"github.com/go-swagger/go-swagger/examples/ShivaniCustomServerExample1/db"
 	// "github.com/segmentio/ksuid"
```

Comment:
please remove commented code all over the palaces.
in case it has some significance then add a comment why??...

Reply:
Okay. Removed the commented lines